[CON-702] Keap guide read/write (#116)
amp-labs/docs: src/provider-guides/keap.mdx

- [hooks](https://developer.infusionsoft.com/docs/rest/#tag/REST-Hooks/operation/list_stored_hook_subscriptions)
- [notes](https://developer.infusionsoft.com/docs/rest/#tag/Note/operation/listNotesUsingGET)
- [opportunities](https://developer.infusionsoft.com/docs/rest/#tag/Opportunity/operation/listOpportunitiesUsingGET)
- [orders](https://developer.infusionsoft.com/docs/rest/#tag/E-Commerce/operation/listOrdersUsingGET) (incremental read, create, no update)
- [products](https://developer.infusionsoft.com/docs/rest/#tag/Product/operation/listProductsUsingGET)
- [subscriptions](https://developer.infusionsoft.com/docs/rest/#tag/E-Commerce/operation/listSubscriptionsUsingGET)  (no update)
- [tag_categories](https://developer.infusionsoft.com/docs/rest/#tag/Tags/operation/createTagCategoryUsingPOST) (no read or update, only create)
- [tags](https://developer.infusionsoft.com/docs/rest/#tag/Tags/operation/listTagsUsingGET) (no update)
- [tasks](https://developer.infusionsoft.com/docs/rest/#tag/Task/operation/listTasksUsingGET) (incremental read)
- [users](https://developer.infusionsoft.com/docs/rest/#tag/Users/operation/listUsersUsingGET) (no update)

> The Keap connector also includes these read-only objects:

Reviewer: ```suggestion
The Keap connector only supports reading from the following objects:
```

- [campaigns](https://developer.infusionsoft.com/docs/rest/#tag/Campaign/operation/listCampaignsUsingGET)
- [commissions](https://developer.infusionsoft.com/docs/rest/#tag/Affiliate/operation/listCommissionsUsingGET) (incremental read)
- [merchants](https://developer.infusionsoft.com/docs/rest/#tag/Merchant/operation/getMerchantAccountsUsingGET)
- [programs](https://developer.infusionsoft.com/docs/rest/#tag/Affiliate/operation/listProgramsUsingGET)
- [redirectlinks](https://developer.infusionsoft.com/docs/rest/#tag/Affiliate/operation/listAffiliateRedirectLinksUsingGET)
- [stage_pipeline](https://developer.infusionsoft.com/docs/rest/#tag/Opportunity/operation/listOpportunityStagePipelinesUsingGET)
- [transactions](https://developer.infusionsoft.com/docs/rest/#tag/E-Commerce/operation/listTransactionsUsingGET) (incremental read)

> Multiple Keap objects support custom fields, which are conveniently returned as native fields.

Reviewer: ```suggestion
Custom fields are returned just like any other field on objects that support them.
```

> If you have experience with the Keap API, you know that there is an additional step required to determine the field name from its field identifier.

Reviewer: ```suggestion
```

> This functionality is provided out of the box, as "native fields".

Reviewer: ```suggestion
```
